plastron/auth.py

```diff
@@ -6,6 +6,7 @@
 import logging
 from requests import Session
 from typing import Dict, List, Type
+from .exceptions import ConfigError
```

Comment:
We are using the absolute import paths elsewhere, so this should be `from plastron.exceptions import ConfigError`.